float32 servo1
float32 servo2
float32 top
float32 bottom